week-2/Homework/mandatory/2-function-creation.js

```diff
@@ -26,7 +44,9 @@ The function must:
 */
 
 function remove(arr, index) {
-  return; // complete this statement
+  let array = arr;
+  arr.splice(index, 1);
```

Comment:
Careful here. As you can see in https://www.w3schools.com/jsref/jsref_splice.asp, the splice method modifies the original array. This is something we do not want according to the requirements. To make this perfect, could you think of a way to make sure we do not modify the original array?